src/widget/emoticonswidget.cpp

```diff
@@ -165,29 +165,23 @@ void EmoticonsWidget::wheelEvent (QWheelEvent *e)
 {
     if (e->orientation() == Qt::Vertical)
     {
-     if (e->delta() < 0 )
-     {
-        stack.setCurrentIndex(stack.currentIndex()+1);
-     }
-     else
-     {
-            stack.setCurrentIndex(stack.currentIndex()-1);
-     }
-    emit PageButtonsUpdate();
+        if (e->delta() < 0)
+            stack.setCurrentIndex(stack.currentIndex() + 1);
+        else
+            stack.setCurrentIndex(stack.currentIndex() - 1);
 
+        emit pageButtonsUpdate();
     }
 }
 
-void EmoticonsWidget::PageButtonsUpdate()
+void EmoticonsWidget::pageButtonsUpdate() const
```

Comment:
I could be wrong, but doesn't this change the object and should therefore not be const?